### DISCUSSION TITLE
Is jellyfin affected by the recent Docker Hub policy change? Is there an alternative registry?

### DISCUSSION THREAD

**Discussion by dscottboggs** [jellyfin/jellyfin#9534] [Q&A] [answered]
Docker [recently revealed](https://blog.alexellis.io/docker-is-deleting-open-source-images/) that the free tier for organizations is going away, except organizations which have been approved under a certain process. It looks like you guys are already registered as one of those approved organizations, but I wanted to check in and make sure that there wouldn't be any issues in a few weeks.

**Comment by nielsvanvelzen** [comment, marked as answer]
From [Reddit](https://www.reddit.com/r/jellyfin/comments/11sqbxt/comment/jcfuz67/):

> 1. We're working on submitting an application as an OSS project and will see how that goes.
> 2. We'll be working on setting up our own Docker container registry using our mirror system.

**Reply by dscottboggs** [reply]
Thanks
